$OpenBSD: patch-src_libs_utils_buildablehelperlibrary_cpp,v 1.3 2018/04/07 07:49:40 rsadowski Exp $

Prefer qmake-qt5 to qmake4 by removing the QDir::Reversed flag.

Index: src/libs/utils/buildablehelperlibrary.cpp
--- src/libs/utils/buildablehelperlibrary.cpp.orig
+++ src/libs/utils/buildablehelperlibrary.cpp
@@ -87,8 +87,7 @@ FileName BuildableHelperLibrary::findSystemQt(const En
                 return FileName::fromString(qmakePath);
         }
 
-        // Prefer qmake-qt5 to qmake-qt4 by sorting the filenames in reverse order.
-        foreach (const QFileInfo &fi, dir.entryInfoList(possibleQMakeCommands(), QDir::Files, QDir::Name | QDir::Reversed)) {
+        foreach (const QFileInfo &fi, dir.entryInfoList(possibleQMakeCommands(), QDir::Files, QDir::Name)) {
             if (fi.fileName() == qmake)
                 continue;
 
